CHANGELOG.md

```diff
@@ -1,6 +1,32 @@
 kOS Mod Changelog
 =================
 
+# v0.18.2
+
+[Insert witty title here :-P]
+------------------------------
+
+### BREAKING CHANGES
+* As usual, you MUST recompile all KSM files before running them on the new version.  Some of the changes have altered how the VM works.
+* Nothing else... we hope.
+
+### NEW FEATURES
+* Compatibility with KSP version 1.0.5
+* `run once ...` syntax to run a script only once per session.
+* Volumes and processors have better integration.
```

Comment:
Once @Dunbaratu has the documentation pages generated, I'm going to link to your documentation pages.  Do you think we need to be more explicit if we have that link?